src/mcp2515.cpp

```diff
@@ -108,46 +108,8 @@ MCP2515::ERROR MCP2515::reset(void) {
 
   delay(10);
 
-  uint8_t zeros[14];
-  memset(zeros, 0, sizeof(zeros));
-  setRegisters(MCP_TXB0CTRL, zeros, 14);
-  setRegisters(MCP_TXB1CTRL, zeros, 14);
-  setRegisters(MCP_TXB2CTRL, zeros, 14);
-
-  setRegister(MCP_RXB0CTRL, 0);
-  setRegister(MCP_RXB1CTRL, 0);
-
   setRegister(MCP_CANINTE, CANINTF_RX0IF | CANINTF_RX1IF | CANINTF_ERRIF | CANINTF_MERRF);
 
-  // receives all valid messages using either Standard or Extended Identifiers that
-  // meet filter criteria. RXF0 is applied for RXB0, RXF1 is applied for RXB1
-  modifyRegister(MCP_RXB0CTRL,
```

Comment:
Good call! I'll try that out next.

Reply:
Wow, using rx buffer 0 saved another 122bytes of flash.

Take a look at the commit because it wasn't as clean as my change to only use tx buffer 0.

The rx side was a little different because the buffer-specific method wasn't doing the status check logic to see if the buffer had any data. I had to move the status check into that method and therefore make it only work for buffer 0 (not that the bootloader cares, but it's just ugly looking code now).